src/main/java/entity/Collectible.java

```diff
@@ -0,0 +1,32 @@
+package entity;
+
+import entity.Item;
+
+public class Collectible extends Item {
+
+    /**
+     * Creates a new entity.Collectible object.
+     * @param collectibleType the type of this entity.Collectible.
+     */
+    public Collectible(String collectibleType){
+        super(collectibleType);
+    }
+
+    /**
+     * Creates a new entity.Collectible object.
```

Comment:
and specify here that this constructor should be used for essence, which can be instantiated with a starting value